packages/salesforcedx-vscode-apex/test/vscode-integration/commands/forceApexExecute.test.ts

```diff
@@ -159,11 +159,9 @@ describe('Force Apex Execute', () => {
 
   describe('Format Execute Anonymous Response', () => {
     let outputStub: SinonStub;
-    let showOutputChannel: SinonStub;
 
```

Comment:
any ideas on how to test the channel service? since showChannelOutput is called by the base class in the utils package now (commandletExecutors.ts), it doesn't seem to be stubbing/testing as expected :/

Reply:
I think I ran into a similar issue with the telemetry service. Have you tried stubbing or spying on the channel output before calling the code in utils ? Do we have "generic" tests for the channel service when it's called by the commandlet code as part of the utils module ?

Reply:
Yeah I tried stubbing/spying on `channelService.showChannelOutput()` but it thinks that the method never gets called so the stub/spy is ineffective. It looks like in the utils module we're using proxyquire to do any testing around channel service, but we also aren't actually testing  `showChannelOutput` directly by stubbing/spying on it- probably for the same reason lol

Reply:
Have you tried stubbing `ChannelService.prototype.showChannelOutput`? Also I think it's not called in `executor.run` here, rather `executor.execute`.